kbase-extension/static/kbase/js/common/cellComponents/tabs/infoTab.js

```diff
@@ -52,7 +52,10 @@ define([
         const model = config.model;
 
         function start(arg) {
-            const appSpec = model.getItem('app.spec');
+            let appSpec = model.getItem('app.spec');    // for plain app cell
```

Comment:
I'm not crazy about how this forks. It's due to how normal app cells vs. the bulk import cell has information.

I also came to realize that the app metric info that gets represented here is stored in the cell at startup time, which means the latest app usage info isn't shown here. IMO, this tab should just take an app id and fetch all of its information on load. This does mean a couple of RPCs, but they should be fairly light weight.